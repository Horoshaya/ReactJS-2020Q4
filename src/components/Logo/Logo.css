.title {
  margin: 0;
  color: #f65261;
  text-align: center;
  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
    "Lucida Sans", Arial, sans-serif;
  font-weight: 100;
}

.titleAccent {
  font-weight: 900;
}
