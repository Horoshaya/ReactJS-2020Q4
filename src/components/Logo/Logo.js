import React from "react";
import styles from "./Logo.css";

const Logo = () => {
  return (
    <h2 className={styles.title}>
      <span className={styles.title}>netflix</span>roulette
    </h2>
  );
};

export default Logo;
