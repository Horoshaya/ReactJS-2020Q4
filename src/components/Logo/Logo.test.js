import React from 'react';
import Logo from './Logo';
import rerender from 'react-test-renderer';

describe('Logo', () => {
  it('snapshot should be created', () => {
    const component = rerender.create(<Logo />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
