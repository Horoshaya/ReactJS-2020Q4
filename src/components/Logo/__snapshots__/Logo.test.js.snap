// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo have text 1`] = `
<h2>
  <span>
    netflix
  </span>
  roulette
</h2>
`;

exports[`Logo snapshot should be created 1`] = `
<h2>
  <span>
    netflix
  </span>
  roulette
</h2>
`;
